github/resource_github_repository_webhook.go

```diff
@@ -14,6 +16,17 @@ func resourceGithubRepositoryWebhook() *schema.Resource {
 		Read:   resourceGithubRepositoryWebhookRead,
 		Update: resourceGithubRepositoryWebhookUpdate,
 		Delete: resourceGithubRepositoryWebhookDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				combined := strings.Split(d.Id(), "/")
```

Comment:
Probably a nitpick, but I this variable deserves a better name, e.g. `parts` or something similar that's plural.